packages/ui/Scrollable/Scrollbar.js

```diff
@@ -22,7 +23,7 @@ const
  * @param {String} value - CSS Variable value.
  */
 const setCSSVariable = (element, variable, value) => {
-	element.style.setProperty(variable, value);
+	ReactDOM.findDOMNode(element).style.setProperty(variable, value); // eslint-disable-line react/no-find-dom-node
```

Comment:
Do we need `findDOMNode` here?

Reply:
Currently, I could not find any solution for it. Let's revisit this with another issue.